hello_network/README.md

```diff
@@ -30,15 +24,15 @@ console to check that your Nerves device indeed has successfully established
 network connectivity and that DNS name resolution works. The expected output
 when DNS resolution is available is something like this:
 
-    iex(1)> HelloNetwork.test_dns
-    {:ok,
-     {:hostent, 'nerves-project.org', [], :inet, 4,
-      [{192, 30, 252, 154}, {192, 30, 252, 153}]}}
-
+``` elixir
+iex(1)> HelloNetwork.test_dns()
+{:ok,
+ {:hostent, 'nerves-project.org', [], :inet, 4,
+  [{192, 30, 252, 154}, {192, 30, 252, 153}]}}
+```
 ## Limitations
```

Comment:
I think we should just remove the whole "Limitations" section.
We can also remove the links at the bottom since they're no longer used in the body (DHCP, SSDP, etc.).